app/controlplane/pkg/data/integrationattachment.go

```diff
@@ -93,28 +93,39 @@ func (r *IntegrationAttachmentRepo) FindByIDInOrg(ctx context.Context, orgID, id
 		return nil, nil
 	}
 
-	return entIntegrationAttachmentToBiz(integration), nil
+	return entIntegrationAttachmentToBiz(integration).IntegrationAttachment, nil
```

Comment:
?

Reply:
`entIntegrationAttachmentToBiz` now returns a `IntegrationAndAttachment` so we need to extract the `IntegrationAttachment`